layers/+lang/ocaml/packages.el

```diff
@@ -82,6 +83,14 @@
       (spacemacs/declare-prefix-for-mode 'tuareg-mode "mh" "help")
       (spacemacs/declare-prefix-for-mode 'tuareg-mode "mr" "refactor"))))
 
+(defun ocaml/post-init-imenu ()
```

Comment:
This function will not be called since `ocaml-packages` doesn't contain `imenu`. You either need to add it or remove this unreachable code.